docs/Deprecating-Disabling-and-Removing-Formulae.md

```diff
@@ -42,16 +44,17 @@ A formula should be disabled to indicate to users that the formula cannot be use
 
 The most common reasons for disabling a formula are:
 
-- it cannot be built from source (meaning no bottles can be built)
+- it cannot be built from source (meaning no new bottles can be built)
 - it has been deprecated for a long time
 - the upstream repository has been removed
```

Comment:
```suggestion
```

This is usually deprecation rather than instant disable as the bottles still work. If we need to build new bottles, the above "cannot be built from source" reason covers this.